git-version

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+v=$(git --version)
+echo $v
+if [[ $v == "git version 2"* ]]; then
+    echo "Found git version > 2.x"
+    node grab-git-info
+else
+    echo "Found git version < 2.x"
```

Comment:
May be git is not available at all; remove this echo.  We already have `echo $v` on top